theme-utils.mjs

```diff
@@ -76,8 +76,8 @@ const commands = {
 	},
 	'version-bump-themes': {
 		helpText:
-			'Bump the version of any theme that has had changes since the last deployment. This includes bumping the version of any parent themes and updating the changelog for the theme.',
-		run: versionBumpThemes,
+			'Bump the version of any theme that has had changes since the last deployment. This includes bumping the version of any parent themes and updating the changelog for the theme. Optionally specify a single theme to version bump.',
+		run: ( args ) => versionBumpThemes(	args?.[ 1 ].split( /[ ,]+/ ) ),
```

Comment:
Because arguments are optional, adding an optional chaining.

```suggestion
		run: ( args ) => versionBumpThemes(	args?.[ 1 ]?.split( /[ ,]+/ ) ),
```